pyface/data_view/api.py

```diff
@@ -0,0 +1,35 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+
+# NOTE: this public-facing API is provisional and may change in future
+# minor releases through until Pyface 8
+
```

Comment:
If this is turned into a module level docstring then it can be visible in the generated API documentation. e.g.
https://github.com/enthought/traitsui/blob/b855634ec98690b84228c7c7cdcd66b1ebe8b2ba/traitsui/testing/api.py

Reply:
(Not a request change, just a comment for consideration.)